docs/deploy.md

```diff
@@ -0,0 +1,24 @@
+# Deploying
+
+If you would like to run your own instance of this plugin, see the [docs for deploying probot plugins](https://github.com/probot/probot/blob/master/docs/deployment.md).
+
+## Setup
+
+```
+# Install dependencies
+npm install
+
+# Run the bot
+npm start
+```
+
+Teacherbot requires these **Permissions & events** for the GitHub App:
+
+- Administration - **Read & Write**
+- Issues - **Read & Write**
```

Comment:
@hectorsector according to teacherbot's app permissions, in addition to the ones I listed here, it also currently has:

-  **Read & Write** permission to Commit Statuses. 
- Is receiving webhooks for **Pull Request Reviews** and **Pull Request Review Comments**
- **Read Only** access to **Repository Contents**.

Based on the functionality being released, it seems like the ones listed above are "extra". Do you agree? Should we try removing them and re-test? I realize teacher-bot in the real world is also checking to see the file is in the _pins directory so I'm guessing some of these permissions are related to that feature.